### PR TITLE
Improved PERMAS export

### PR DESCRIPTION
**Pull request by nilswagner**
Better export in case of multiple element types.

### CODE REVIEW

File: meshio/permas_io.py [hunk 1]
```diff
@@ -132,55 +132,79 @@ def write(
...
+        #
+        # Avoid non-unique element numbers in case of multiple element types by num_ele !!!
+        #
+        num_ele = 0
+        for eltyp in cells.keys(): # Loop over available element types
+            
+            if eltyp == 'line':
+                fh.write('!\n')
+                fh.write('        $ELEMENT TYPE = PLOTL2 ESET = PLOTL2\n')
+                cell = cells[eltyp]
```
Comment (nschloe) [inline meshio/permas_io.py]: `cell` can be retrieved by `for elem_type, cell in cells.iteritems()` above.

File: meshio/permas_io.py [hunk 2]
```diff
@@ -132,55 +132,79 @@ def write(
...
+                for k, c in enumerate(cell):
+                    n = len(c)
+                    form = '        %8d ' + ' '.join(n * ['%8d']) + '\n'
+                    fh.write(form % (k+1, c[0]+1, c[1]+1))
+                num_ele += len(cell)
+
+            elif eltyp == 'quad':
```
Comment (nschloe) [inline meshio/permas_io.py]: All of the `if-else` clauses are equal except for the `$ELEMENT TYPE` string and the number of format values. The first can be put into a dictionary, the second concatenated into one tuple with `num_ele+k+1`, and we don't need this huge repetition anymore.

**Comment by nschloe** [comment]
Can we close this one?

**Comment by nilswagner** [comment]
Yes.
Am 03.12.2015 22:38 schrieb "Nico Schlömer" <notifications@github.com>:

**Pull request closed without merging**